### PR TITLE
Settings to constants

### LINKED ISSUE #59: Consolidate Source class Settings - Niche
Move all settings for a `Source` to a consolidated location to make updates easier to manage and make sure to not forget updates to any one of the values:
- **Phoenix campaign signup**: https://github.com/DoSomething/mbc-user-import/blob/master/src/MBC_UserImport_Source_Niche.php#L243-L246
- **Mobile Commons**: https://github.com/DoSomething/mbc-user-import/blob/master/src/MBC_UserImport_Source_Niche.php#L329-L347

### PR DESCRIPTION
**Pull request by DeeZone**
Fixes #59 

Move add adjustable settings to constants at the top of the class:
- `mb-niche-welcome_new-new_v1-0-0`
- `mb-niche-welcome_existing-new_v1-0-0`
- `mb-niche-welcome_existing-existing_v1-0-0`
- `206777` // Birthday Mail - Mobile Commins opt_in
- `2461` // Birthday Mail Phoenix

### CODE REVIEW

**Comment by DeeZone** [comment]
@jamjensen Please review these changes to support moving Niche user imports to Shower Songs.

File: src/MBC_UserImport_Source_Niche.php [hunk 1]
```diff
@@ -11,6 +11,12 @@
 class MBC_UserImport_Source_Niche extends MBC_UserImport_BaseSource
 {
 
+  const WELCOME_EMAIL_NEW_NEW = 'mb-niche-welcome_new-new_v1-1-0';
+  const WELCOME_EMAIL_EXISTING_NEW = 'mb-niche-welcome_existing-new_v1-1-0';
+  const WELCOME_EMAIL_EXISTING_EXISTING = 'mb-niche-welcome_existing-existing_v1-1-0';
+  const MOBILE_COMMONS_SIGNUP = 207563; // Birthday Mail
```
Comment (jamjensen) [inline src/MBC_UserImport_Source_Niche.php]: The opt-in path id is correct, but it refers to Shower Songs, not Birthday Mail. Can we update the text?

File: src/MBC_UserImport_Source_Niche.php [hunk 2]
```diff
@@ -11,6 +11,12 @@
...
+  const PHOENIX_SIGNUP = 2590; // Birthday Mail
```
Comment (jamjensen) [inline src/MBC_UserImport_Source_Niche.php]: The campaign nid for Shower Songs is 3590, not 2590.

**Comment by jamjensen** [comment]
👍

**Pull request merged**